src/views/quiz-settings/quiz-settings.js

```diff
@@ -0,0 +1,114 @@
+class QuizSettings {
+    quizAbout;
+    questionsNum;
+    questionsType;
+
+    static createRadioButton(val,nameOfRadio,what,side){
+        const div=document.createElement('div');
+        const button = document.createElement('input');
+        button.setAttribute('id',val);
+        button.type='radio';
+        button.value=val;
+        button.name=nameOfRadio;
+        const label=document.createElement('label');
+        label.setAttribute('for',val);
+        label.setAttribute('class',side);
+        label.innerText=val;
+        button.addEventListener('click', () => {
+            if (what=="about"){
+                this.quizAbout=val;
+            }
+            if (what=="type"){
+                this.questionsType=val;
+            }
+            
+        });
```

Comment:
Extract handler to separate function to improve readability. Use strict equality.